tests/unit/controllers/v0.3/AddressValidationAPI.test.js

```diff
@@ -337,27 +321,6 @@ describe("AddressValidationAPI", () => {
       });
     });
 
-    it("returns a 'valid-address' response but card denied if address is outside NY", () => {
```

Comment:
We are no longer denying accounts if a patron is outside NY. They just get a temporary account.